src/three-components/ARRenderer.js

```diff
@@ -279,6 +296,34 @@ height: 100%;`);
     }
   }
 
+  /**
+   * It appears that XRSession's `inputsourceschange` event is not implemented
+   * in Chrome Canary as of m72 for 'screen' inputs, which would be preferable
+   * since we only need an "select" event, rather than track a pose on every
+   * frame (like a 6DOF controller). Due to this bug, on every frame, check to see
+   * if an input exists.
+   * https://bugs.chromium.org/p/chromium/issues/detail?id=913703
+   */
+  [$processXRInput](frame) {
+    const {session} = frame;
+
+    // Get current input sources. For now, only 'screen' input is supported,
+    // which is only added to the session's active input sources immediately
+    // before `selectstart` and removed immediately after `selectend` event.
+    // If we have a 'screen' source here, it means the output canvas was tapped.
+    const sources = session.getInputSources().filter(
+        input => input.targetRayMode === 'screen');
+
+    if (sources.length === 0) {
+      return;
+    }
+
+    const pose = frame.getInputPose(sources[0], this[$frameOfReference])
```

Comment:
So if I understand this correctly, `screen` input sources are ephemeral and depend on actual input to show up in the list, so if one does it implies there was some touch input to select a placement?

Reply:
Correct; although when held, the input will exist until release, so existence doesn't necessarily imply 'touchstart'